### ISSUE TITLE
Amplify CLI `amplify configure` does not work on Ubuntu for Windows

### ISSUE THREAD

**Issue by jkeys-nmsu** [aws-amplify/amplify-cli#531]
**Describe the bug**
`amplify configure` does not work as intended on Ubuntu for Windows

**To Reproduce**
1. Setup Ubuntu (with Windows Subsystem for Linux) for Windows
2. Install Amplify and Amplify CLI
3. Amplify init your new project
4. Add resources (in this case, auth, hosting, and analytics
5. run `$amplify configure`

**Expected behavior**
A configuration prompt that I can walk through.

**Screenshots**
![image](https://user-images.githubusercontent.com/22964702/49395747-99590000-f6f4-11e8-8a33-ce6a0bb8e45d.png)
![image](https://user-images.githubusercontent.com/22964702/49395796-b392de00-f6f4-11e8-9bee-e2b4fa56cfb2.png)

**Desktop (please complete the following information):**
 - OS: Windows 10, Ubuntu on Windows

**Additional context**
The Amplify folder is in the Windows filesystem but is accessed through Ubuntu's /mnt/c/ directory.

**Comment by yuth** [comment]
To understand the problem, you were able to configure amplify first time and initialize the project. You got the above error when you tried to run `amplify configure`.

Could you also include the version of `node` and `amplify CLI` You can use the following commands to get the version
```bash
$ node -v 
$ amplify -v 
```

**Comment by jkeys-ecg-nmsu** [comment]
Yes. I just would like the aws_exports file so that I don't have to manually Amplify.configure(). Here's my node and Amp CLI versions:

```
$node -v
v8.11.3
$amplify -v
0.1.34
```

**Comment by yuth** [comment]
> Yes. I just would like the aws_exports file so that I don't have to manually Amplify.configure()

`amplify configure` command is used to set up credentials that are used to make requests for setting up the resources. Is that what you want to do? 

To get the `aws-exports.js` you need to push your resources to cloud by runnung `amplify push`

**Comment by Aeolun** [comment]
Issue is still there. Exactly the same issue when running `amplify configure` in WSL.

**Comment by jkeys-ecg-nmsu** [comment]
@kaustavghosh06

**Comment by wenzaca** [comment]
Ensure that the Node Version is above 8.x as required in the documentation.

For AWS AMI run: 
curl -sL https://rpm.nodesource.com/setup_10.x | sudo bash -
sudo yum install nodejs
